open-machine-learning-jupyter-book/data-science/working-with-data/non-relational-data.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>There's a missing "." before "Each."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Sorry, I can't understand this.